docs/guides/qedma-qesem.ipynb

```diff
@@ -37,40 +37,56 @@
    "id": "74823696",
```

Comment:
<p>Having an example where <code>stds</code> includes values &gt; 1 reads strangely. Was that intentional? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
That was a typo. Fixed